release 7.0.14 (#146)
FamilySearch/GEDCOM.io: _pages/changelog.md

- Changes anticipating a coming extension registry:

>     - Add URIs for sets of enumeration values. This has changes some fragment identifiers in the HTML version of the spec and could cause hotlinks to the specific sections discussing enumeration sets to change.

Author: ```suggestion
    - Add URIs for sets of enumeration values. This changed some fragment identifiers in the HTML version of the spec and could cause hotlinks to the specific sections discussing enumeration sets to change.
```

        - `mul` can be used if there is no single primary language, but is unlikely to provide practical functionality beyond `und`.
        - `zxx` can be used for ASCII art and other non-language text, and can improve accessibility for screen readers.

> - Clarify that empty *payloads* are encoded as missing `LineVal`s and empty `LineVal`s are not been permitted; this has been true since 7.0.0 but was easily overlooked in the previous text.

Author: ```suggestion
- Clarify that empty *payloads* are encoded as missing `LineVal`s and empty `LineVal`s are not permitted; this has been true since 7.0.0 but was easily overlooked in the previous text.
```

# Version 7.0.0

> As a major release and the first update to the specification in 20 years, there a many changes in this version.

Author: ```suggestion
As a major release and the first update to the specification in 20 years, there are many changes in this version.
```

## New Extensibility

> - Every standard tag now has a single "default" meaning, even if it also has additional meanings in other contexts. Tags conforming to this default meaning can now be used by extensions as substructures of structures with extention tags.

Author: ```suggestion
- Every standard tag now has a single "default" meaning, even if it also has additional meanings in other contexts. Tags conforming to this default meaning can now be used by extensions as substructures of structures with extension tags.
```

- Age phrases are now only phrases, not long-hand terms for specific age ranges

> - Previously registered values (APPROVED_SYSTEM_ID, RECEIVING_SYSTEM_NAME, etc) are kept as-is if present; new ones are URIs instead of having a separate registration process

Author: ```suggestion
- Previously registered values (APPROVED_SYSTEM_ID, RECEIVING_SYSTEM_NAME, etc.) are kept as-is if present; new ones are URIs instead of having a separate registration process
```

- Dual-year dates were used with widely different semantics and have been replaced by more flexibility in date phrases.

> - `SEX` is now unambiguously biological sex at birth; all other related concepts (gender identity, sexual preference, sex reassignment, etc) are time-varying attributes and to be stored in an individual attribute instead

Author: ```suggestion
- `SEX` is now unambiguously biological sex at birth; all other related concepts (gender identity, sexual preference, sex reassignment, etc.) are time-varying attributes and to be stored in an individual attribute instead
```
